koschei/db.py

```diff
@@ -129,7 +129,8 @@ def get_sessionmaker():
     if __sessionmaker:
         return __sessionmaker
     __sessionmaker = sessionmaker(bind=get_engine(), autocommit=False,
-                                  class_=KoscheiDbSession, query_cls=Query)
+                                  class_=KoscheiDbSession, query_cls=Query,
+                                  expire_on_commit=False)
```

Comment:
How about just adding new function `commit_without_expire` and use it only where it's necessary? If not then at least providing `commit_with_expire` for convenience.